internal/config/docs.toml

```diff
@@ -430,6 +430,14 @@ MemThreshold = '4gb' # Default
 # GoroutineThreshold is the maximum number of actively-running goroutines the node can spawn before profiling begins.
 GoroutineThreshold = 5000 # Default
 
+[Pyroscope]
+# ServerAddress sets the address that will receive the profile logs. It enables the profiling service.
+ServerAddress = '' # Default
+# AuthToken sets the needed Auth Token on Server Addresses that require an Auth Token.
+AuthToken = '' # Default
+# Environment sets the target environment tag in which profiles will be added to.
+Environment = 'mainnet' # Default
```

Comment:
Could we include `# Example`s for the ones that have no default?
Is the `Environment` tag meaningful? or just a free-form user field?

Reply:
@jmank88 the `Environment` field is a free-form user field which will allow us to categorize whether this runs on mainnet, or some specific network, or if it is integration-tests from CI. I do not expect NOPs to set this value themselves, unless they are instructed to do it.

I would expect infra, QA and us setting this value to capture profiling information.